README.md

```diff
@@ -25,27 +29,30 @@ HER is friends with HEAR and HERE.
 
 HERE is friends with THERE.
 
-As before, HI is in its own social network. 
+As before, HI is in its own social network.
 ```
 1 + 1 + (1+ 1) + (1 +  1) +  1    = 7
 HI  HE  SHE HER   HEAR HERE  THERE
 ```
- 
+
 Please note that we want to see production quality code! Be sure to include any and all tests and
```

Comment:
I would add "You can write it in any programming language. We would use Ruby, but you don't have to!".